test/ios/getBuildProperty.spec.js

```diff
@@ -0,0 +1,19 @@
+const chai = require('chai');
+const expect = chai.expect;
+const xcode = require('xcode');
+const getBuildProperty = require('../../src/ios/getBuildProperty');
+
+const project = xcode.project('test/fixtures/project.pbxproj');
+
+describe('ios::getPlistPath', () => {
+
+  beforeEach(() => {
+    project.parseSync();
+  });
+
+  it('should return build property from main target', () => {
+    const plistPath = getBuildProperty(project, 'INFOPLIST_FILE');
+    expect(plistPath).to.equals('"Basic/Info.plist"');
```

Comment:
Note it returns Basic/Info.plist, not BasicTest/Info.plist and that's the purpose of this test-case.